docs/smart_stack.md

```
@@ -0,0 +1,85 @@
+
+- `cortex/stacks.json` — Stack definitions  
+- `cortex/stack_manager.py` — Stack manager class  
+- `cortex/cli.py` — CLI command handler  
+- `test/test_smart_stacks.py` — Tests for stack loading, GPU detection, and dry-run
+
+---
+
+## Demo Video
+
+Video walkthrough:  
+https://drive.google.com/file/d/1MkXEWCsVUmbzXbKyKnETfNaOZKZ3mnld/view?usp=sharing
```

markdownlint-cli2 (0.18.1)

Bare URL used

(MD034, no-bare-urls)